### PR TITLE
(CDAP-1204) Refactor QueueConstants

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

**Comment by anwar6953** [comment]
LGTM

File: cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java [hunk 1]
```diff
@@ -112,8 +112,7 @@ protected HBaseQueueAdmin(Configuration hConf,
     this.cConf = cConf;
     this.tableUtil = tableUtil;
     // todo: we have to do that because queues do not follow dataset semantic fully (yet)
-    String unqualifiedTableNamePrefix =
-      type == QUEUE ? QueueConstants.QUEUE_TABLE_PREFIX : QueueConstants.STREAM_TABLE_PREFIX;
+    String unqualifiedTableNamePrefix = type.toString();
```
Comment (jwang47) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java]: Maybe add a `getPrefix()` method, or at least somehow have some common code to transform `QueueType` into the table name prefix.

Comment (chtyim) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java]: There is a separate effort right now to introduce a common abstract base for QueueAdmin (CDAP-1177), I am leaving it to there.

File: cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java [hunk 2]
```diff
@@ -430,7 +430,7 @@ public Long getResult() {
    * @return constructed table name
    */
   public static String getStateStoreTableName(Id.Namespace namespace) {
-    return String.format("cdap.%s.%s.state.store", namespace.getId(), QueueConstants.STREAM_TABLE_PREFIX);
+    return String.format("cdap.%s.%s.state.store", namespace.getId(), QueueConstants.QueueType.STREAM.toString());
```
Comment (albertshau) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java]: need to lowercase otherwise it will be different

Comment (albertshau) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java]: o nevermind, just saw the change below

Comment (anwar6953) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java]: It seems like it won't be.
+    QUEUE("queue"),
+    STREAM("stream");
toString returns that value.

Comment (anwar6953) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java]: k

**Comment by jwang47** [comment]
LGTM

**Comment by chtyim** [comment]
Buidl pass http://builds.cask.co/browse/CDAP-DUT878-1

**Pull request merged**